administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -473,7 +569,7 @@ protected function downloadPackage($url, $target)
     {
         try {
             Log::add(Text::sprintf('COM_JOOMLAUPDATE_UPDATE_LOG_URL', $url), Log::INFO, 'Update');
-        } catch (\RuntimeException $exception) {
+        } catch (RuntimeException $exception) {
```

Comment:
```suggestion
        } catch (\RuntimeException $exception) {
```
Revert this change because in J4 the file is namespaced.